### PR TITLE
WRN-3815: Migrate EditableIntegerPicker unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist
* [X] I have read and understand the contribution guide
* [ ] A CHANGELOG entry is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Migrated tests from Enzyme to Testing Library.

### Resolution
Changed test structure to work with Testing Library.

### Links
WRN-3815

### Comments

### CODE REVIEW

File: EditableIntegerPicker/tests/EditableIntegerPicker-specs.js [hunk 1]
```diff
@@ -1,258 +1,269 @@
-import {mount} from 'enzyme';
-import {EditableIntegerPicker, EditableIntegerPickerBase} from '../EditableIntegerPicker';
 import Spotlight from '@enact/spotlight';
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+
```
Comment (daniel-stoian-lgp) [inline EditableIntegerPicker/tests/EditableIntegerPicker-specs.js]: remove empty line

File: EditableIntegerPicker/tests/EditableIntegerPicker-specs.js [hunk 2]
```diff
@@ -1,258 +1,269 @@
...
-			{attachTo: node}
-		);
+		setTimeout((done) => {
+			const expected = 'input';
+			const actual = screen.getByLabelText('undefined change a value with left right button').children.item(1);
 
-		picker.find('PickerItem').simulate('click', {target: {type: 'click'}});
+			expect(actual).toHaveClass(expected);
+			done();
+		}, 300);
```
Comment (daniel-stoian-lgp) [inline EditableIntegerPicker/tests/EditableIntegerPicker-specs.js]: I tried to do a screen.debug() or  a screen.getByRole that would throw an error 100% and the test didn't fail. that means the code inside setTimeout() is never called, that means the expect function is not verified. Can you check again? I don't think we need the setTimeout()

File: EditableIntegerPicker/tests/EditableIntegerPicker-specs.js [hunk 3]
```diff
@@ -1,258 +1,269 @@
...
 
-			picker.simulate('keyDown', {keyCode: 50});
-			const expected = 1;
-			const actual = picker.find('input').length;
-
-			expect(actual).toBe(expected);
-		}
-	);
+			expect(actual).toHaveClass(expected);
+			done();
```
Comment (daniel-stoian-lgp) [inline EditableIntegerPicker/tests/EditableIntegerPicker-specs.js]: here again. the code inside settimeout is never called. Maybe Mikyung's solution can be applied here

**Review by daniel-stoian-lgp — CHANGES_REQUESTED** [review]

File: EditableIntegerPicker/tests/EditableIntegerPicker-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply EditableIntegerPicker/tests/EditableIntegerPicker-specs.js]: Done

File: EditableIntegerPicker/tests/EditableIntegerPicker-specs.js (hunk 2 above)
Comment (alexandrumorariu) [reply EditableIntegerPicker/tests/EditableIntegerPicker-specs.js]: Changed them to work properly, thanks for the catch.

File: EditableIntegerPicker/tests/EditableIntegerPicker-specs.js (hunk 3 above)
Comment (alexandrumorariu) [reply EditableIntegerPicker/tests/EditableIntegerPicker-specs.js]: Changed them to work properly, thanks for the catch.

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**